edgelessdb/getting-started/marblerun.md

```diff
@@ -1,44 +1,78 @@
 # Running EdgelessDB under MarbleRun
 
-If you want to run EdgelessDB as a distributed application in a confidential cluster, you can combine it with [MarbleRun](https://marblerun.sh).
+To run EdgelessDB as a service in a confidential cluster, combine it with [MarbleRun](https://marblerun.sh).
 
-## Changes
-When you are running EdgelessDB as a Marble, keys and credentials will be managed by MarbleRun. EdgelessDB will no longer generate its own root certificate nor its sealing key. Instead, they will be provided by MarbleRun's [Secrets management](https://docs.edgeless.systems/marblerun/#/content/features/secrets-management). The root certificate for EdgelessDB needs to be explicitly defined in MarbleRun's manifest. We explain this in more detail in the next section below. Given now MarbleRun will handle EdgelessDB's certificate and key management, EdgelessDB's own recovery method will be unavailable. Instead, MarbleRun will handle recovery for your entire cluster.
+When running EdgelessDB as a Marble, secrets will be [managed by MarbleRun](https://docs.edgeless.systems/marblerun/#/features/secrets-management). EdgelessDB will no longer generate its own root certificate nor its sealing key. The root certificate for EdgelessDB needs to be defined in MarbleRun's manifest. Furthermore, EdgelessDB's own recovery method will be unavailable. MarbleRun will handle recovery for your entire cluster.
 
-## Setup
-### Launch the MarbleRun Coordinator
-Setup and launch the MarbleRun Coordinator, by following the [Deployment guide](https://docs.edgeless.systems/marblerun/#/content/deployment/cloud?id=deploy-marblerun) until it is waiting for a manifest to be uploaded.
+## Extend the MarbleRun manifest
+To add EdgelessDB to your MarbleRun cluster, add to the [MarbleRun manifest](https://docs.edgeless.systems/marblerun/#/workflows/define-manifest)
+* the `edgelessdb` package
+* an encryption key `edb_masterkey`
```

Comment:
Maybe use camelCase here instead of snake_case? Looks a bit out of place in the manifest in my opinion.

Reply:
I agree about aesthetics, but wanted to be consistent with our docs and existing samples that use snake_case for secrets.